test/services/human_time_service_test.dart

```diff
@@ -0,0 +1,116 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/services/human_time_service.dart";
+
+import "../mocks/providers/provider_human_time_service.dart";
+
+void main() {
+  group("Human Time Service Unit tests", () {
+    late HumanTimeService humanTimeService;
+
+    setUp(() {
+      // Setup the `HumanTimeService` for constant time
+      humanTimeService = HumanTimeService(
+        currentDateTimeCallback: constantTimeCallback(),
+      );
+    });
+
+    test("Check correct simple absolute time", () {
+      // Setup actual values for simple date
+      final time1 = DateTime.utc(2002);
+
+      const expectedSimpleHumanText = "Tuesday, January 1, 2002 00:00";
+
+      // Use the service to get human time
+      final actualSimpleHumanText = humanTimeService.textTimeAbsolute(
+        time1,
+      );
+
+      // Check that the absolut time value is correct
+      expect(
+        actualSimpleHumanText,
+        expectedSimpleHumanText,
+      );
+    });
+
+    test("Check correct complex absolute time", () {
+      // Setup actual values for complex specific date
+      final time2 = DateTime.utc(
+        2023,
+        11,
+        21,
+        1,
+      );
+
+      const expectedComplexHumanText = "Tuesday, November 21, 2023 01:00";
+
+      // Use the service to get human time
+      final actualComplexHumanText = humanTimeService.textTimeAbsolute(
+        time2,
+      );
+
+      // Check that the specific absolut time value is correct
+      expect(
+        actualComplexHumanText,
+        expectedComplexHumanText,
+      );
+    });
+
+    test("Check special case 'now' relative time", () {
+      // Setup actual relative values for 'now' case
+      final nowTime = constantTestingTime.add(const Duration(seconds: 1));
+
+      const expectedNowRelativeTime = "now";
+
+      // Use the service to get relative human time
+      final actualNowTimeHumanText = humanTimeService.textTimeSince(
+        nowTime,
```

Comment:
In my opinion, "nowTime" can be a confusing variable name, because it's suggest that it's the actual time (something similar to DateTime.now).  Do you think we should rename it (something like "justBeforeTime")?

Reply:
Ok, done in 9507b49